src/components/NavDrawer/navDrawer.js

```diff
@@ -92,13 +92,13 @@ export default function PersistentDrawerLeft(params) {
     const redirect = (buttonName) => {
         //TODO: remove below line; line exists to suppress warning due to currently unused 'openSearch' state
         if (openSearch === true)
-
+            console.log()
```

Comment:
without this line code does not seems to be working.

Reply:
The search button doesn't currently do anything. I'm keeping the if condition in order to make use of the openSearch state. Otherwise, we get compilation warnings for unused states. Also, the switch statement should NOT be inside the if condition. These are separate. This:
![image](https://user-images.githubusercontent.com/72574688/196060263-c235982f-f400-4d7c-bf93-52e30940eea8.png)

should be:
![image](https://user-images.githubusercontent.com/72574688/196060303-145b130f-71a0-47bd-b581-72b4202b540f.png)

Also, please remove the TODO's for the Home and Profile routes since you already added them. Thanks!

Reply:
My guess to the issue you were facing (i.e. routing not working properly) was the formatting. With the current formatting in the code, if the switch block is right below the if condition (i.e. no empty line between the if condition and switch block), the switch block is part of the if condition, and as such is not functional since openSearch is set to false by default. Fixing the formatting should get it working properly. Let me know if that fixes it on your end.